feat(docs): new linking guide (#18935)
expo/expo: docs/pages/guides/deep-linking.md

---
title: Deep linking
---

import { ConfigReactNative } from '~/components/plugins/ConfigSection';
import { Collapsible } from '~/ui/components/Collapsible';
import { Terminal } from '~/ui/components/Snippet';

It is often desirable for regular HTTPS links (without a custom URL scheme) to directly open your app on mobile devices. This allows you to send notification emails with links that work as expected in a web browser on desktop, while opening the content in your app on mobile. iOS refers to this concept as "universal links" while Android calls it "deep links" (but in this section, we are specifically discussing deep links that do not use a custom URL scheme).

> > The Expo SDK also supports deferred deep links with the [Branch API](../versions/latest/sdk/branch.md).

Author: ```suggestion
> The Expo SDK also supports deferred deep links with the [Branch API](/versions/latest/sdk/branch).
```

> > The Expo SDK also supports deferred deep links with the [Branch API](https://github.com/expo/config-plugins/tree/main/packages/react-native-branch).

Author: ```suggestion
> Deferred deep links can be implemented with [`react-native-branch`](https://github.com/expo/config-plugins/tree/main/packages/react-native-branch).
```

It is often desirable for regular HTTPS links (without a custom URL scheme) to directly open your app on mobile devices. This allows you to send notification emails with links that work as expected in a web browser on desktop, while opening the content in your app on mobile. iOS refers to this concept as "universal links" while Android calls it "deep links" (but in this section, we are specifically discussing deep links that do not use a custom URL scheme).

> > The Expo SDK also supports deferred deep links with the [Branch API](/versions/latest/sdk/branch).

Reviewer: ```suggestion
> The Expo SDK also supports deferred deep links with the [Branch API](https://github.com/expo/config-plugins/tree/main/packages/react-native-branch).
```

This link no longer works, so I think we can link to the config plugin Readme here. I did the same for https://docs.expo.dev/guides/linking/ recently.
